library/lua/3rdparty/alt_getopt.lua

```diff
@@ -57,11 +60,16 @@ local function get_opt_map(opts)
     return options
 end
 
+---@param opt string|integer
```

Comment:
opt can only be a string at this point (but I see that the more permissive type might be needed because it can be string|integer in other contexts)

Reply:
As above, I believe.